Tests/LibThreading/TestThread.cpp

```diff
@@ -4,13 +4,30 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <AK/Time.h>
 #include <LibTest/TestCase.h>
 #include <LibThreading/Thread.h>
 #include <unistd.h>
 
+using namespace AK::TimeLiterals;
```

Comment:
Defaulting to this entire namespace feels a bit heavy to me just for the single `_ms` instance. Why not:

```cpp
using AK::TimeLiterals::operator""_ms;
```

Reply:
I personally find the import-single-user-literal syntax horrid and actively avoid it :upside_down_face: 

More seriously, this is an inline namespace, which exists to avoid the weight of something like `using namespace std;`. So all we get here are a couple of literals, and this is how we import these everywhere else in the codebase.

Reply:
`AK::TimeLiterals` is pretty small - agreed! This form of `using` is used elsewhere in the codebase (probably not for `_ms` though, as you stated) but if you're opposed, let's drop it now :^)